GherkinSyncTool/CommandLineOptions/CommandLineParserHelper.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using CommandLine;
+using CommandLine.Text;
+using NLog;
+
+namespace GherkinSyncTool.CommandLineOptions
+{
+    public static class CommandLineParserHelper
+    {
+        private static readonly Logger Log = LogManager.GetLogger(MethodBase.GetCurrentMethod()?.DeclaringType?.Name);
+        private static readonly string Heading = $"{Program.Version}Quantori GherkinSyncTool is an open-source console " +
```

Comment:
```suggestion
        private static readonly string Heading = $"{Program.Version} Quantori GherkinSyncTool is an open-source console " +
```

Reply:
`Program.Version` has a line brake, so it looks ok.
![image](https://user-images.githubusercontent.com/24970976/140026396-34dd0e00-0b5a-4bef-a0c2-6a97ea62a91b.png)